src/components/settings/owner/AddOwnerDialog/index.tsx

```diff
@@ -39,17 +40,21 @@ export const AddOwnerDialog = () => {
 
   return (
     <Box paddingTop={2}>
-      <div>
-        <Track {...SETTINGS_EVENTS.SETUP.ADD_OWNER}>
-          <Button
-            onClick={() => setOpen(true)}
-            variant="text"
-            startIcon={<SvgIcon component={AddIcon} inheritViewBox fontSize="small" />}
-          >
-            Add new owner
-          </Button>
-        </Track>
-      </div>
+      <CheckWallet>
+        {(isOk) => (
+          <Track {...SETTINGS_EVENTS.SETUP.ADD_OWNER}>
+            <Button
```

Comment:
Same here with the Tooltip position. Maybe we should give the `Track` wrapper `display: inline-block`
<img width="836" alt="Screenshot 2023-03-02 at 10 44 18" src="https://user-images.githubusercontent.com/5880855/222391842-c19425fc-ae1b-4a5d-b5c5-93ae063c8f81.png">

Reply:
I made it a span by default.